MaxText/configs/base.yml

```diff
@@ -44,20 +44,20 @@ int8_training: False
 # Global parameter scale needs to be a power of 2. If you want finer grained control of the model sizes 
 # then you should explicitly set base_embed_dim, base_num_heads, base_mlp_dim, base_num_decoder_layers and/or head_dim.
 global_parameter_scale: 1
-base_emb_dim: 2560
-base_num_heads: 8
-base_mlp_dim: 8192
-base_num_decoder_layers: 16
-head_dim: 256
+base_emb_dim: 4096
+base_num_heads: 32
+base_mlp_dim: 11008
+base_num_decoder_layers: 32
+head_dim: 128
 # activation functions are .
-mlp_activations: ["relu"]
+mlp_activations: ["silu"]
 dropout_rate: 0
```

Comment:
There is another norm config eps to be added. As we should rename layernorm etc, I can address this issue in another PR.

Reply:
There is a epsilon param in the LayerNorm computation, would that suffice?
https://github.com/google/maxtext/blob/main/MaxText/layers.py#L635

Reply:
It's different from llama2's config, their default eps is 1e-5
https://huggingface.co/meta-llama/Llama-2-7b-hf/blob/main/config.json#L18

Reply:
Let's not change in the layerNorm class, we need rename this class to RmsNorm